.github/ISSUE_TEMPLATE/BUG-REPORT.yaml

```diff
@@ -0,0 +1,39 @@
+name: Bug Report
+description: File a Corso bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: textarea
+    id: version
+    attributes:
+      label: Corso Version?
+      description: What version of Corso are you running?
```

Comment:
We can maybe provide the command here to get the corso version (`corso --version`).

Reply:
Done.